content/usecaseItems.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
After some googling, I find the easiest and most clean way to fetch data and feed them to the component would be using a md file including a list of items, rather than retrieving the yaml file and looping through all the information from individual md files (or maybe there is a nice way to do this but I did not find). We can keep it for now as it is always nice to have an overview of all use cases, and change it when it is necessary.